docs/index.rst

```diff
@@ -480,19 +480,32 @@ overwrite the global `opentracing.tracer` reference.
 
 Typically this method looks something like::
 
-    from opentracing.scope_managers import ThreadLocalScopeManager
-    from ddtracer.opentracer import Tracer
+    from ddtrace.opentracer import Tracer, set_global_tracer
 
     def init_tracer(service_name):
+        """
+        Initialize a new Datadog opentracer and set it as the
+        global tracer.
+
+        This overwrites the opentracing.tracer reference.
+        """
         config = {
           'agent_hostname': 'localhost',
           'agent_port': 8126,
-          'debug': False,
-          'enabled': True,
-          'global_tags': {},
         }
-        return Tracer(service_name, config=config, scope_manager=ThreadLocalScopeManager)
+        tracer = Tracer(service_name, config=config)
+        set_global_tracer(tracer)
```

Comment:
we should check what are the choices made here: https://github.com/opentracing/opentracing-python/pull/95